### PR TITLE
add http support

### LINKED ISSUE #8: Validate for http url
At present it only searches for `https` url and fails for `http` url.
It should work for both http and https

### PR DESCRIPTION
**Pull request by michaelymo**
Fixes #8 

test log
```
$ cat .git/config | grep git
        url = http://github.com/michaelymo/flip-remote.git
$ yarn test
yarn run v1.13.0
$ node index.js
✔ Github Remote Flipped
✨  Done in 0.15s.
$ cat .git/config | grep git
        url = git@github.com:michaelymo/flip-remote.git
```

Thanks for attaching a tag of 'good first Issue' to the issue!
I'm new to contribution :)

### CODE REVIEW

**Comment by michaelymo** [comment]
Thanks for review.
Yes. I will change it now.

**Comment by michaelymo** [comment]
changed.
but I'm worry about variable name of RegExp Object.

(if I should do this comment before the 4188 commit, please teach me.

**Comment by arshadkazmi42** [comment]
There is no issues with order of comments here.

File: index.js [hunk 1]
```diff
@@ -14,14 +14,13 @@ const flip = async (currentPath) => {
     // Read config file content
     let configContent = await readConf(currentPath);
 
-    const httpsPathRegex = new RegExp(GH_PATH.HTTPS_PATH, 'i');
+    const httpsPathRegex = new RegExp('https?://github.com/', 'i');
```
Comment (arshadkazmi42) [inline index.js]: I think variable name `httpPathRegex` will be better.

This looks like a new constant.
Let's move it to the [gh-conf-constants](https://github.com/gh-conf/gh-conf-constants) repository.
I have created an issue for this here [gh-conf/gh-conf-constants#3](https://github.com/gh-conf/gh-conf-constants/issues/3)

Would you like to work on it?

Comment (michaelymo) [reply index.js]: I didn't know that it is the library(gh-conf-constants) you are participating in 😯

Yes! I'll do it!

**Comment by michaelymo** [comment]
Thanks for merge and release!
I have updated this.

**Comment by michaelymo** [comment]
Thank you for all that you are to me.
I have got great experience!

**Pull request merged**